### PR TITLE
Fix `mouse`

### PR DESCRIPTION
**Pull request by inancgumus**
## What?

See #1317

## Checklist

- [x] I have performed a self-review of my code
- [x] I have added tests for my changes
- [x] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: tests/mouse_test.go [hunk 1]
```diff
@@ -5,30 +5,104 @@ import (
...
+
+		// Simulate a click at the button coordinates
+		box := button.BoundingBox()
+		m.Click(box.X, box.Y, nil)
+
+		// Verify the button's text changed
+		assert.Equal(t, "Clicked!", button.TextContent())
+	})
+
+	t.Run("double_click", func(t *testing.T) {
```
Comment (ankur22) [inline tests/mouse_test.go:36]: I think this test should also ensure that a click has occurred twice. There are two things that need to occur:

1. `ondblclick` event is fired.
2. the element is clicked twice.

Before https://github.com/grafana/xk6-browser/pull/1121, the `ondblclick` event was fired, but the element wasn't being clicked on twice. That's why there was a counter in the test to ensure that the element was clicked on twice.

So we need to add something into the test to ensure that the element is in fact being clicked on twice too.

**Review by ankur22 — APPROVED** [review]
LGTM.

I think the `dblclick` test needs to be amended though.

File: tests/mouse_test.go (hunk 1 above)
Comment (inancgumus) [reply tests/mouse_test.go:36]: 👍 [8beba15](https://github.com/grafana/xk6-browser/pull/1318/commits/8beba159aaf0f124f72c45404bd3035a3d38cc8e)

**Review by ankur22 — APPROVED** [review]
LGTM 🚀

**Pull request merged**